iqrs_method.py

```diff
@@ -31,9 +32,8 @@ def lambda_handler(event, context):
             input_data,
             produce_columns("movement_", questions_list.split(',')),
             produce_columns("iqrs_", questions_list.split(',')),
-            event["distinct_values"].strip().split(',')
+            distinct_values
```

Comment:
I think this would look better if all the `.split(',')` sections were together, either in the `# Environment variables` section or the `calc_iqrs` call.

Reply:
The next commit should address that query